numpy/core/tests/test_umath_accuracy.py

```diff
@@ -5,13 +5,15 @@
 import pytest
 from ctypes import c_float, c_int, cast, pointer, POINTER
 from numpy.testing import assert_array_max_ulp
+from numpy.core._multiarray_umath import __cpu_features__
 
-runtest = sys.platform.startswith('linux') and (platform.machine() == 'x86_64')
+IS_AVX = __cpu_features__.get('AVX512F') or \
+        (__cpu_features__.get('FMA3') and __cpu_features__.get('AVX2'))
+runtest = sys.platform.startswith('linux') and IS_AVX
 platform_skip = pytest.mark.skipif(not runtest,
                                    reason="""
-                                   stick to x86_64 and linux platforms.
-                                   test seems to fail on some of ARM and power
-                                   architectures.
+                                   testing only the AVX2+FMA3 and AVX-512F
+                                   implementations on linux platforms.
```

Comment:
maybe "avoid testing inconsistent platform library implementations"

Reply:
Sounds good to me.